pkg/kubelet/dockertools/docker.go

```diff
@@ -211,16 +210,12 @@ func (p throttledDockerPuller) IsImagePresent(name string) (bool, error) {
 	return p.puller.IsImagePresent(name)
 }
 
-// TODO (random-liu) Almost never used, should we remove this?
-// DockerContainers is a map of containers
-type DockerContainers map[kubetypes.DockerID]*docker.APIContainers
-
-func (c DockerContainers) FindPodContainer(podFullName string, uid types.UID, containerName string) (*docker.APIContainers, bool, uint64) {
-	for _, dockerContainer := range c {
+// An internal helper function.
+func findPodContainer(dockerContainers []*docker.APIContainers, podFullName string, uid types.UID, containerName string) (*docker.APIContainers, bool, uint64) {
```

Comment:
We should move it to the test file (docker_test.go) since this function is used only in tests.